server/server.js

```diff
@@ -3,9 +3,11 @@ const express = require("express");
 const { join } = require("path");
 const { log, expressPino } = require("./logger");
 const { port } = require("./config");
-const { sync, database } = require("./models");
+const { sequelize } = require("./gql/models");
 const { stopNotifications } = require("./configNotifiers/configNotifierCreator");
-const router = require("./routes");
+const serverApi = require("./routes");
+
+const applyExecutableMiddleware = require("./playground");
```

Comment:
At first glance, this line of code has very little meaning to me. I'm wondering why is there a file called `playground` and why does it export a function called `applyExecutableMiddleware`, and what on earth is an executable middleware?

If we leave this function named as `applyExecutableMiddleware` then I fear the term 'executable middleware' will be casually thrown around in conversation all the time when referring to a specific instance of the core execution layer (especially when we start dealing with multiple instances at the same time). To me that term makes no sense and would be confusing to someone without the proper context.

I would suggest we find a better name for the file and for the function within.

I know it's a mouthful but perhaps something like `applyExecutionLayer` or `applyExecutionLayerForSchema` (it is an execution layer for a specific schema after all). With this kind of naming, a newcomer would be asking _what's an execution layer?_ and that's a good question to ask.